lib/project/results-pane.coffee

```diff
@@ -140,3 +149,19 @@ class ResultsPaneView extends ScrollView
       @removeClass('no-results')
     else
       @addClass('no-results')
+
+  collapseAllResults: =>
+    i=0
```

Comment:
I took a look and I feel like these would belong more in `results-view`.  Definitely take my advice with a grain of salt though - I've never really touched find-and-replace before.